src/Components/v2/ArtworkFilter/ArtworkFilters/MediumFilter.tsx

```diff
@@ -3,7 +3,7 @@ import React, { FC } from "react"
 import { useFilterContext } from "../ArtworkFilterContext"
 
 interface Props {
-  mediums: Array<{
+  mediums?: Array<{
```

Comment:
This should probably just be optional, since it's checked w/ the default list of hardcoded mediums used when not present a few lines below.